source/Mlos.Agent.Server/MlosAgentServer.cs

```diff
@@ -214,5 +219,17 @@ public static void Main(string[] args)
 
             Console.WriteLine("Mlos.Agent exited.");
         }
+
+        /// <summary>
+        /// The command line options for this application.
+        /// </summary>
+        private class CliOptions
+        {
+            [Option("executable", Required = false, Default = null, HelpText = "A path to an executable to start (e.g. 'target/bin/Release/SmartCache').")]
+            public string Executable { get; set; }
+
+            [Option("optimizer-uri", Required = false, Default = null, HelpText = "A URI to connect to the MLOS Optimizer service over GRPC (e.g. 'http://localhost:50051').")]
+            public Uri OptimizerUri { get; set; }

```

Comment:
Future PR: add the ability to specify the location of the SettingsRegistry's here on the CLI instead of (or in addition to?) as the `MLOS_SETTINGS_REGISTRY_PATH` environment variable